src/packages/client/src/runtime/transformDmmf.ts

```diff
@@ -119,7 +119,10 @@ function transformInputTypes(document: DMMF.Document): DMMF.Document {
 
           // there might not be an equals field, if it's a relation filter
           if (equalsField) {
-            f.inputType.unshift(equalsField.inputType[0])
+            if (equalsField.inputType[0].type !== 'Json') {
```

Comment:
Maybe we could use a comment here as it's hard to process everything that's happening in this file 😄

Reply:
Ok, will add commments :)